ci-operator/step-registry/upi/deprovision/azure/upi-deprovision-azure-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+function queue() {
+    local TARGET="${1}"
+    shift
+    local LIVE
+    LIVE="$(jobs | wc -l)"
+    while [[ "${LIVE}" -ge 45 ]]; do
+    sleep 1
+    LIVE="$(jobs | wc -l)"
+    done
+    echo "${@}"
+    if [[ -n "${FILTER:-}" ]]; then
+    "${@}" | "${FILTER}" >"${TARGET}" &
+    else
+    "${@}" >"${TARGET}" &
+    fi
+}
+
+set +e
+export PATH=$PATH:/tmp/shared
+
+echo "Deprovisioning cluster ..."
+if [[ -f ${ARTIFACT_DIR}/installer/metadata.json ]]; then
```

Comment:
I think the metadata.json is still not available.
The [output from the upi-deprovision-azure step](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/22320/rehearse-22320-pull-ci-openshift-installer-master-e2e-azure-upi/1468957454816514048/artifacts/e2e-azure-upi/upi-deprovision-azure/build-log.txt) is the following, where I would expect some output from the `az group delete` command.
```
Deprovisioning cluster ...
```

Reply:
Isn't this a "make sure it's deleted" step? If ipi-deprovision works then shouldn't the metadata.json disappear?

Reply:
It is a "make sure it's deleted" step in the sense that the resource group should be deleted by the ipi-deprovision step. However, the ipi-deprovision step will not delete the metadata.json file from the shared directory.